rollup.config.mjs

```diff
@@ -14,19 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { computeBanner } from './scripts/shared/banner.mjs';
-// generate warning when running with Node 18
-// (node:75278) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
-import pkg from './package.json' assert { type: 'json' };
-
+import commonjs from '@rollup/plugin-commonjs'; // at least, needed to bundle mxGraph which is only available as a CommonJS module
+import resolve from '@rollup/plugin-node-resolve';
 import terser from '@rollup/plugin-terser';
 import cleanup from 'rollup-plugin-cleanup';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
**question:** can you explain in the ignore rule directive why it is not resolved?
We know it works when the rollup is running, so I don't understand why there would be a problem that would force us to disable the eslint rule.

Reply:
```suggestion
import terser from '@rollup/plugin-terser';
import cleanup from 'rollup-plugin-cleanup';
// eslint-disable-next-line import/no-unresolved -- The package is malformed. So, the rule doesn't resolve it.
```

Reply:
If I write `import externals from 'rollup-plugin-node-externals/dist';`, this rule works. But, in my memories, there is another problem after ^^

Reply:
Fine, the proposed description is clear.